lib/model/CMetricModel.cc

```diff
@@ -379,15 +381,22 @@ bool CMetricModel::computeProbability(const std::size_t pid,
         }
     }
 
-    if (pJoint.empty()) {
+    if (pJoint.empty() && skippedResults == false) {
```

Comment:
As per above, maybe keep this together with other checks below, which it relates to.